### DISCUSSION TITLE
Getting last show status change

### DISCUSSION THREAD

**Discussion by jonathanantoine** [trakt/trakt-api#353] [Q&A] [answered]
Hello,

Is there a way to get last show changes of status ?
My users would like to have a list of recently change of status of their shows.

Thanks.

**Comment by rectifyer** [comment, marked as answer]
The `/updates` method contains any shows where the status changed. Can you provide more details on how you'd expect this new API method to work? We don't keep a history of status changes currently.

**Reply by jonathanantoine** [reply]
That's what I was looking for. 
Ideally, the caller would provide a list of show IDs and get the last changes for a provided list of properties (title, status, etc.) And as a result you would get the last changes made with previous and new value.

Thanks for answering.

**Reply by rectifyer** [reply]
We unfortunately don't store a history of the changed metadata like TMDB has. If you need that, might be best to get it directly from TMDB? We also don't have general API methods that accept a list of show IDs since that would effectively break caching and be unique for nearly every user.
